<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="frame">
          <property name="text">
           <string>Frame</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="currentFrame">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="slash">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalFrames">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="addFrame">
        <property name="text">
         <string>Add frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteFrame">
        <property name="text">
         <string>Delete Frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="globalChanges">
        <property name="text">
         <string>Global changes</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1294</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="separator"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSaveFileAs"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionNextFrame"/>
    <addaction name="actionPreviousFrame"/>
    <addaction name="separator"/>
    <addaction name="actionAddFrame"/>
    <addaction name="actionDeleteFrame"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelectionMode"/>
   <addaction name="actionPolylineMode"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSelectionMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SelectionMode</string>
   </property>
  </action>
  <action name="actionPolylineMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PolylineMode</string>
   </property>
   <property name="toolTip">
    <string>PolylineMode</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveFileAs">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionNextFrame">
   <property name="text">
    <string>Next Frame</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionPreviousFrame">
   <property name="text">
    <string>Previous Frame</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionAddFrame">
   <property name="text">
    <string>Add Frame</string>
   </property>
   <property name="toolTip">
    <string>Add Frame</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionDeleteFrame">
   <property name="text">
    <string>Delete Frame</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
